src/transformers/file_utils.py

```diff
@@ -1229,12 +1224,16 @@ def hf_bucket_url(
         filename = f"{subfolder}/{filename}"
 
     if mirror:
-        endpoint = PRESET_MIRROR_DICT.get(mirror, mirror)
+        logger.warning(
+            "Deprecated: The mirroring is deprecated."
+            "Tuna and BFSU mirrors are no longer available."
+            "We will soon provide a self-hosted alternative."
+        )
```

Comment:
Please use a proper deprecation warning here like in other parts of the codebase:
```
warnings.warn(..., FutureWarning)
```
as a log can be silenced and will be displayed every time this code is executed.